sunpy/visualization/wcsaxes_compat.py

```diff
@@ -124,14 +129,21 @@ def default_wcs_grid(axes):
     if y.coord_type != 'latitude':
         y.set_coord_type('latitude')
 
-    x.set_major_formatter('s.s')
-    y.set_major_formatter('s.s')
+    if units[0] is u.deg:
+        x.set_major_formatter('dd')
+    else:
+        x.set_major_formatter('s.s')
+
+    if units[1] is u.deg:
+        y.set_major_formatter('dd')
+    else:
+        y.set_major_formatter('s.s')
 
     axes.coords.grid(color='white', alpha=0.6, linestyle='dotted',
                      linewidth=0.5)
 
 
-def wcsaxes_heliographic_overlay(axes):
+def wcsaxes_heliographic_overlay(axes, grid_spacing=10*u.deg):
```

Comment:
ah,... maybe don't because you can put a float... Wait, it's the docstring incorrect?